### DISCUSSION TITLE
Rust Buildpack Maintainers

### DISCUSSION THREAD

**Discussion by ryanmoran** [paketo-buildpacks/feedback#5] [Q&A] [answered]
Hi folks! We are going to be starting the process of implementing the Rust Buildpack RFC. This will mean that we want to create a maintainer team to handle maintenance of the buildpacks as they move through incubation in the `paketo-community` org. The RFC outlines that @dmikusa will act as one of those maintainers, but we would like, with Dan's approval, to find a second maintainer to ensure that we keep up with the PR review process.

If you are interested in becoming a maintainer for the Rust buildpacks, please leave a comment on this topic and let us know. We will hopefully be able to find someone by the next working group meeting on 2/2/21.

**Comment by dmikusa** [comment]
This is fantastic! Excited to get started.

**Comment by ForestEckhardt** [comment, marked as answer]
I would be more than happy to sign on as a maintainer for the Rust buildpack!
